src/components/Navbar/index.tsx

```diff
@@ -34,24 +34,24 @@ export type NavBarProps = {
   site: string;
   utmContext: string;
   SearchComponent?: JSX.Element;
-  LogoComponent?: JSX.Element;
+  LogoComponent?: () => JSX.Element;
 };
 
+const DefaultLogo = () => (
+  <img
+    className="fdsNavbar-navLogo"
```

Comment:
To note: we only need braces (`{}`) when you need to do templating, e.g. 

```
className={`hello ${world}`)
```

If we don't have any template, we can just use a regular string:


```
className='hello'
```

I know you're just following the pattern in this file, just wanted to give a heads up. We should update this file later.